### ISSUE TITLE
Integration of matrixprofile with tsfresh

### ISSUE THREAD

**Issue by vanbenschoten** [blue-yonder/tsfresh#785]
Moving earlier conversations with @nils-braun , @tylerwmarrs and @MaxBenChrist to GitHub.

**Overview**
Our goal is to create a combiner that 1) calculates the matrix profile (with supplied parameter k) for each time series dimension, then 2) returns Tukey's 5 number summary for each dimension. Note that additional features can be easily added in the future as the matrix profile will already be calculated for the dataset.

**Technical Notes**
The integration will most likely use the matrixprofile.compute functionality. Parameters for compute are listed below - there are several values we'll need to align on, and whether these should be user-supplied or leverage sensible defaults (window size, sample pct, etc).

The package also needs to be hosted on conda to be a non-optional dependency. This workstream is being handled [here](https://github.com/matrix-profile-foundation/matrixprofile/issues/70).

_compute parameters_

        ts (array_like) – The time series to analyze.

        windows (int, array_like) – The window(s) to compute the MatrixProfile. Note that it may be an int for a single matrix profile computation or an array of ints for computing the pan matrix profile.

        query (array_like, optional) – The query to analyze. Note that when computing the PMP the query is ignored!

        sample_pct (float, default 1) – A float between 0 and 1 representing how many samples to compute for the MP or PMP. When it is 1, the exact algorithm is used.

        threshold (float, default 0.98) – The correlation coefficient used as the threshold. It should be between 0 and 1. This is used to compute the upper window size when no window(s) is given.

        n_jobs (int, default = 1) – Number of cpu cores to use.

**Comment by nils-braun** [comment]
Would be very happy to see this :-) (and it would also finally start to solve other long outstanding issues, e.g. https://github.com/blue-yonder/tsfresh/issues/658)

We did discuss in private communication already some thoughts, let me quickly summarize them:
* it will be a combiner feature
* you have already mentioned the conda "issue" - perfect!
* I am currently facing problems when calling `mp.compute` in one of the robot example time series. I have created an issue https://github.com/matrix-profile-foundation/matrixprofile/issues/71

I will open a PR with a first draft, then we can iterate on the code.
